recipes/reprimand/meta.yaml

```diff
@@ -0,0 +1,94 @@
+source:
+  url: https://github.com/wokast/RePrimAnd/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d72d4749d3c2087f9a231023d406bd4c30ea3c953d2a9f0a6c7780168d6c8d6d
+  
+build:
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - meson 
+    - ninja
+    - pkg-config
+  host:
+    - boost
+    - gsl
+    - libopenblas
+    - hdf5
+    - zlib
+  run:
+    - gsl
+    - libopenblas
+    - hdf5
+
+test:
+  imports:
+
+  requires:
+
+  commands:
+    - test -f $PREFIX/lib/libRePrimAnd$SHLIB_EXT    
+    - test -f $PREFIX/include/reprimand/c2p_report.h                
+    - test -f $PREFIX/include/reprimand/eos_barotr_spline.h      
+    - test -f $PREFIX/include/reprimand/hydro_prim.h
+    - test -f $PREFIX/include/reprimand/con2prim_imhd.h            
+    - test -f $PREFIX/include/reprimand/eos_barotr_table.h       
+    - test -f $PREFIX/include/reprimand/interpol.h
+    - test -f $PREFIX/include/reprimand/con2prim_imhd_internals.h   
+    - test -f $PREFIX/include/reprimand/eos_hybrid.h             
+    - test -f $PREFIX/include/reprimand/interpol_linear.h
+    - test -f $PREFIX/include/reprimand/config.h                    
+    - test -f $PREFIX/include/reprimand/eos_idealgas.h           
+    - test -f $PREFIX/include/reprimand/interpol_logspl.h
+    - test -f $PREFIX/include/reprimand/datastore.h                 
+    - test -f $PREFIX/include/reprimand/eos_thermal_file.h       
+    - test -f $PREFIX/include/reprimand/interpol_pchip_spline.h
+    - test -f $PREFIX/include/reprimand/eos_barotr_file.h           
+    - test -f $PREFIX/include/reprimand/eos_thermal_file_impl.h  
+    - test -f $PREFIX/include/reprimand/interpol_regspl.h
+    - test -f $PREFIX/include/reprimand/eos_barotr_file_impl.h      
+    - test -f $PREFIX/include/reprimand/eos_thermal.h            
+    - test -f $PREFIX/include/reprimand/intervals.h
+    - test -f $PREFIX/include/reprimand/eos_barotr_gpoly.h          
+    - test -f $PREFIX/include/reprimand/eos_thermal_impl.h       
+    - test -f $PREFIX/include/reprimand/smtensor.h
+    - test -f $PREFIX/include/reprimand/eos_barotropic.h            
+    - test -f $PREFIX/include/reprimand/eos_thermal_internals.h  
+    - test -f $PREFIX/include/reprimand/spherical_stars.h
+    - test -f $PREFIX/include/reprimand/eos_barotropic_impl.h       
+    - test -f $PREFIX/include/reprimand/global_registry.h        
+    - test -f $PREFIX/include/reprimand/star_seq_file.h
+    - test -f $PREFIX/include/reprimand/eos_barotropic_internals.h  
+    - test -f $PREFIX/include/reprimand/hdf5store.h              
+    - test -f $PREFIX/include/reprimand/star_sequence.h
+    - test -f $PREFIX/include/reprimand/eos_barotr_poly.h           
+    - test -f $PREFIX/include/reprimand/hydro_atmo.h             
+    - test -f $PREFIX/include/reprimand/unitconv.h
+    - test -f $PREFIX/include/reprimand/eos_barotr_pwpoly.h         
+    - test -f $PREFIX/include/reprimand/hydro_cons.h
```

Comment:
This is probably an overkill. I'd keep just the first check.

Reply:
I agree. The install process is straightforward copying and the most likely problem would be issues for the whole install folder.